src/Mount.cpp

```diff
@@ -206,6 +210,41 @@ void Mount::readPersistentData()
     LOG(DEBUG_INFO, "[MOUNT]: EEPROM: DEC limits read as %l -> %l", _decLowerLimit, _decUpperLimit);
 }
 
+/////////////////////////////////
+//
+// configureHemisphere
+//
+/////////////////////////////////
+void Mount::configureHemisphere(bool inNorthern, bool force)
+{
+    if ((inNorthernHemisphere != inNorthern) || force)
+    {
+        bool wasTracking = isSlewingTRK();
+        LOG(DEBUG_ANY, "[SYSTEM]: Hemisphere changed (or forced update) to %s.", inNorthern ? "northern" : "southern");
+        LOG(DEBUG_ANY, "[SYSTEM]: Stopping all steppers.");
+        stopSlewing(ALL_DIRECTIONS | TRACKING);
+        waitUntilStopped(ALL_DIRECTIONS);
+        inNorthernHemisphere = inNorthern;
+        bool invertDir       = inNorthernHemisphere ? (RA_INVERT_DIR == 1) : (RA_INVERT_DIR != 1);
```

Comment:
`bool invertDir = inNorthernHemisphere == RA_INVERT_DIR`

Reply:
I did this because I wanted to be sure that it will work since inNorthernHermisphere is a bool, but RA_INVERT_DIR is '0' or '1'... I don't think that 'true == 1' is true.

Reply:
well in worst case we could static cast ... but i think it is fine this way